website/cue/reference/components/sources/base/amqp.cue

```diff
@@ -95,6 +100,12 @@ base: components: sources: amqp: configuration: {
 					}
 				}
 			}
+			desc_file: {
```

Comment:
Sorry, I didn't review this before but this `desc_file` option should be nested under a `protobuf` key like the `gelf` and `json` options you see here, are.

Reply:
Opened https://github.com/vectordotdev/vector/pull/18111 to move it.